test/cmactest.c

```diff
@@ -70,6 +70,79 @@ static struct test_st {
         18,
         "65c11c75ecf590badd0a5e56cbb8af60"
     },
+    // for aes-128-cbc
```

Comment:
Nit: `//` comments are not allowed. Use the regular C comments.

I am slightly worried we do not exercise the burst fully. Could we for example repeat the input in the test buffer multiple times so the input is bigger than the burst buffer? Of course precompute the expected mac value with original CMAC_Update code.

Reply:
I've modified comments.
I've updated test cases, with package size bigger than burst buffer.